.github/workflows/run_tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.10", "3.11"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
pyproject.toml actually has: `requires-python = ">=3.10,<3.13"`
Can drop `3.13` here?

Reply:
or we lift the restriction at the package level ofc.

Reply:
(resolved via following second route as I saw now)